rfcs/0000-dao-deposit-withdraw/0000-dao-deposit-withdraw.md

```diff
@@ -0,0 +1,401 @@
+---
+Number: "0000"
+Category: Standards Track
+Status: Proposal
+Author: Jan Xie, Xuejie Xiao, Ian Yang
+Organization: Nervos Foundation
+Created: 2019-10-30
+---
+
+# Deposit and Withdraw in Nervos DAO
+
+## Abstract
+
+This document describes deposit and withdraw transaction in Nervos DAO.
+
+Note the very end of this RFC will include a `Gotchas` section, including common and very important points you should be aware to use Nervos DAO well without losing CKBs. Please pay attention to this section even if you might want to skip some part of this RFC.
+
+## Motivation
+
+Nervos DAO is a smart contract, with which users can interact the same way as any smart contract on CKB. One function of Nervos DAO is to provide an dilution counter-measure for CKByte holders. By deposit in Nervos DAO, holders get proportional secondary rewards, which guarantee their holding are only affected by hardcapped primary issuance as in Bitcoin.
+
+Holders can deposit their CKBytes into Nervos DAO at any time. Nervos DAO deposit is a time deposit with a minimum deposit period (counted in blocks). Holders can only withdraw after a full deposit period. If the holder does not withdraw at the end of the deposit period, those CKBytes should enter a new deposit period automatically, so holders’ interaction with CKB could be minimized.
+
+## Background
+
+CKB's token issuance curve consists of two components:
+
+- Primary issuance: Hardcapped issuance for miners, using the same issuance curve as Bitcoin, half at every 4 years.
+- Secondary issuance: Constant issuance, the same amount of CKBytes will be issued at every epoch, which means secondary issuance rate approaches zero gradually over time. \[Because epoch length is dynamically adjusted\](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0020-ckb-consensus-protocol/0020-ckb-consensus-protocol.md), secondary issuance at every block is a variable. 
```

Comment:
```suggestion
- Secondary issuance: Constant issuance, the same amount of CKBytes will be issued at every epoch, which means secondary issuance rate approaches zero gradually over time. [Because epoch length is dynamically adjusted](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0020-ckb-consensus-protocol/0020-ckb-consensus-protocol.md), secondary issuance at every block is a variable. 
```